iOS Example Frame/iOS Example FrameUITests/XCUIApplication+Extension.swift

```diff
@@ -61,13 +61,18 @@ extension XCUIApplication {
         //   and hope it is found
         if !keys[key].exists {
             // !!! Shift is not found under keys, but under buttons !!!
-            buttons["shift"].tap()
+            if buttons["shift"].exists {
+                buttons["shift"].tap()
+            }
+            if keys["Shift"].exists {
+                keys["Shift"].tap()
+            }
```

Comment:
I seem to be unable to suggest, but we need to update the comment and maybe make sure both situations cannot occur?

```swift
/// Shift can be found as a key, as well as a button, depending on simulator and OS
if buttons["shift"].exists {
    buttons["shift"].tap()
} else if keys["Shift"].exists {
    keys["Shift"].tap()
}
```